connect/2.1/Dockerfile

```diff
@@ -23,4 +24,5 @@ RUN docker-maven-download debezium mongodb "$DEBEZIUM_VERSION" "$MONGODB_MD5" &&
     docker-maven-download debezium oracle "$DEBEZIUM_VERSION" "$ORACLE_MD5" && \
     docker-maven-download debezium-additional db2 db2 "$DEBEZIUM_VERSION" "$DB2_MD5" && \
     docker-maven-download debezium-additional vitess vitess "$DEBEZIUM_VERSION" "$VITESS_MD5" && \
+    docker-maven-download debezium-optional connect-rest-extension "$DEBEZIUM_VERSION" "$KCRESTEXT_MD5" && \
```

Comment:
This will fail as debezium-optional method assumes the target artifact is a tar archive

https://github.com/debezium/container-images/blob/main/connect-base/2.1/docker-maven-download.sh#L55


@jpechane  just to confirm, should there be a tar for connect-rest-extension?

Reply:
tar was no planned but thinking about that let's do it in the same way as `debezium-scripting` so it should provide tar too.

Reply:
@dude0001   based on Jiri's comment we should keep this and the extension in ``Dockerfile.local`` the way it is now. Tomorrow I will add the appropriate profile to the rest-extension module in main repository.

Reply:
I refactor my logic including assuming there will be a .tar.gz artifact available for the REST API Extension. Please take another look.